oauthproxy.go

```diff
@@ -570,15 +593,15 @@ func (p *OAuthProxy) SignInPage(rw http.ResponseWriter, req *http.Request, code
 	err := p.ClearSessionCookie(rw, req)
 	if err != nil {
 		logger.Printf("Error clearing session cookie: %v", err)
-		p.ErrorPage(rw, http.StatusInternalServerError, "Internal Server Error", err.Error())
+		p.ErrorPage(rw, req, http.StatusInternalServerError, "Internal Server Error", err.Error())
```

Comment:
Just an FYI - Since you mentioned potentially giving clients a better error message page with a more descriptive message:

I've leaned toward keeping externally facing error page messages intentionally vague (while backend logs are rich) or in order to not leak too many details to malicious attackers probing the system.

Hence nearly all these at the moment say "Internal Server Error"